### ISSUE TITLE
Namespace value resources do not convey multiple encoding types

### ISSUE THREAD

**Issue by abargnesi** [OpenBEL/openbel-api#88]
Example: http://next.belframework.org/api/namespaces/sp/Q7Z5M6/equivalents/hgnc

The response does not allow for multiple encoding types even though it likely codes for a gene, RNA, or protein. Also the *type* name is generic. Maybe name it *encoding_type*.

**Response:**

```json
{
  "namespace_value_collection": [
    {
      "namespace_value": {
        "type": "ProteinConcept",
        "identifier": "19837",
        "name": "ABHD12B",
        "title": "abhydrolase domain containing 12B",
        "species": "9606"
      },
      ...snip...
    }
  ]
}
```

**Comment by abargnesi** [comment]
Related to #30.

**Comment by abargnesi** [comment]
Fixed. Included in OpenBEL API 0.4.0.
